app/router.js

```diff
@@ -9,18 +9,18 @@ const Router = EmberRouter.extend(googlePageview, {
 
 Router.map(function() {
   this.route('courses', function() {
-    this.route('id', {path: '/:courseId'});
+    this.route('id', {path: '/:course_id'});
```

Comment:
Why?

Reply:
Alright, I see. Refactoring the params in the child routes corresponding to these routes will solve the problem I think. Will push a commit!

Reply:
Why do you want to refactor this in the first place?

Reply:
the linting calls for snake_case usage in params instead of camelCase, hence fails the test.

Reply:
Done!